proto/hypertrace/agent/config/v1/config.proto

```diff
@@ -98,20 +98,24 @@ message DataCapture {
 
 // PropagationFormat represents the propagation formats supported by agents
 enum PropagationFormat {
+
+    // Default to none. Agent will use its default propagation format
+    PROPAGATION_FORMAT_UNSPECIFIED = 0;
```

Comment:
This is also a breaking change, the agents will need to add the default Propagation format. Should I add an exception for this as well? 
And the name needs to be prefixed for this as in one package, there cannot be more than one enum with the same name. And UNSPECIFIED is used in all enums.

Reply:
we shouldn't change this.  Can you ignore these in the linting rules also.

Reply:
Sure